<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>二级文件系统@涂远鹏</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_Cmd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1011</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="title">
     <string>程序输出</string>
    </property>
    <widget class="QTextBrowser" name="CmdWin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>991</width>
       <height>371</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="cursorWidth">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>1001</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="title">
     <string>快捷输入</string>
    </property>
    <widget class="QGroupBox" name="groupBox_Create">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>5.fcreate创建新文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_CreateFileName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CreateMode">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_CreateFileName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件名/路径：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_CreateMode">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>创建方式：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Create">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+5</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Mkdir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>7.mkdir创建目录</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_MkdirFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_MkdirFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>目录名/路径：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Mkdir">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Ls">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>30</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>3.ls文件列表</string>
     </property>
     <widget class="QPushButton" name="pushButton_Ls">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>显示当前目录</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+3</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Delete">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>8.fdelete删除文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_DeleteFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_DeleteFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件名/路径：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Delete">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+8</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Cd">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>290</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>9.cd打开目录</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_CdFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_CdFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件名/路径：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cd">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+9</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Write">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>421</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>2.fwrite写入文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_WriteToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_WriteFileName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_WriteToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>文件操作句柄：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_WriteFileName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>写入的文件名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_WriteSize">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_WriteSize">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>写入的数据大小：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Write">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+2</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Read">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>431</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>1.fread读取文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_ReadToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ReadFileName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_ReadToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件操作句柄：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ReadFileName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>输出的文件名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ReadSize">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_ReadSize">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>写入的数据大小：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Read">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+1</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Exit">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>290</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>12.exit</string>
     </property>
     <widget class="QPushButton" name="pushButton_Exit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+=</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Fformat">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>290</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>11.fformat格式化</string>
     </property>
     <widget class="QPushButton" name="pushButton_Fformat">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>文件格式化</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+-</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Open">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>160</y>
       <width>281</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>6.fopen 打开文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_OpenFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_OpenMode">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_OpenFilePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件名/路径：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_OpenMode">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>打开方式：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Open">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+6</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Close">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>290</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>10.fclose关闭文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_CloseToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_CloseToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件句柄：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Close">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Seek">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>431</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>4.flseek写入文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_SeekToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_SeekStart">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_SeekToken">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件操作句柄：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_SeekStart">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>起始位置：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_SeekOffset">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_SeekOffset">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>偏移量：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Seek">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+4</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Complete">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="title">
     <string>完整指令输入</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Complete">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Complete">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_AutoTest">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="text">
     <string>自动测试</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+A, Ctrl+T</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
